schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -275,6 +275,18 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-30/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_python2Packages" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-06-30/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_runbooks" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-10-01/Microsoft.Media.json#/resourceDefinitions/mediaServices" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2018-07-01/Microsoft.Media.json#/resourceDefinitions/mediaServices" },
```

Comment:
>mediaServices [](start = 126, length = 13)

by "case" I mean, here you have capital S, but in the schema file you have "mediaservices" where everything is lower case.

Reply:
I checked the existing 2015-10-01 schema and the "mediaservices" (all lower case) version was a mistake.  I have fixed everything to be mixed case ("mediaServices") in the schema file.